### PR TITLE
[Admin][ProductAssociationType] Implement create and update pages

### PR DESCRIPTION
**Pull request by NoResponseMate**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | bootstrap-admin-panel                  |
| Bug fix?        | no                                                       |
| New feature?    | yes                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no |
| Related tickets | -                      |
| License         | MIT                                                          |

### CODE REVIEW

**Review by mpysiak — APPROVED** [review]

File: src/Sylius/Bundle/AdminBundle/Resources/views/Shared/Helper/translations.html.twig [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {% macro default(translationFields, body = null, options = {}) %}
-    <div class="accordion" id="{{ options.accordionId }}">
+    {% set accordionId = options.accordionId|default('translations') %}
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Shared/Helper/translations.html.twig:2]:
```suggestion
    {% set accordion_id = options.accordionId|default('translations') %}
```

File: src/Sylius/Bundle/AdminBundle/Resources/views/product_association_type/form/translations.html.twig [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% import '@SyliusAdmin/Shared/Helper/translations.html.twig' as _translations %}
+
+<div class="bg-white mb-3">
+    {% set body %}
+        {% verbatim %}
+            <div class="row"">
+                <div class="col-12">
+                    {{ form_row(translationForm.name, sylius_test_form_attribute('name', locale)) }}
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Resources/views/product_association_type/form/translations.html.twig:8]: We should slightly refactor translations to add hook for every field, it should be done in a separate PR for all resources already implemented

**Review by GSadee — APPROVED** [review]

File: src/Sylius/Bundle/AdminBundle/Resources/views/product_association_type/form/translations.html.twig (hunk 2 above)
Comment (NoResponseMate) [reply src/Sylius/Bundle/AdminBundle/Resources/views/product_association_type/form/translations.html.twig:8]: Agreed
